index.html

```diff
@@ -1401,6 +1401,7 @@ <h2>Definition of Roles</h2>
 				<p>An input that allows for user-triggered actions when clicked or pressed. See related <rref>link</rref>.</p>
 				<p>Buttons are mostly used for discrete actions. Standardizing the appearance of buttons enhances the user's recognition of the <a>widgets</a> as buttons and allows for a more compact display in toolbars.</p>
 				<p>Buttons support the optional <a>attribute</a> <sref>aria-pressed</sref>. Buttons with a non-empty <sref>aria-pressed</sref> attribute are toggle buttons. When <sref>aria-pressed</sref> is <code>true</code> the button is in a "pressed" <a>state</a>, when <sref>aria-pressed</sref> is <code>false</code> it is not pressed. If the attribute is not present, the button is a simple command button.</p>
+                <p>Authors MAY set <pref>aria-setsize</pref> and <a>aria-posinset</a> on elements with this role when they are <a>accessibility children</a> of <rref>toolbar</rref> with only elements with a <rref>group</rref> role intervening.</p>
```

Comment:
Poop, yeah, totally my fault since that is what I meant.

That being said, if you want to do that on all the roles (and can get to it before me, which is likely), then go for it.